fairing/preprocessors/function.py

```diff
@@ -36,6 +40,9 @@ def __init__(self,
             path_prefix=path_prefix,
             input_files=input_files)
 
+        if notebook_util.is_in_notebook():
```

Comment:
if notebook_util.is_in_notebook(): -> if not notebook_util.is_in_notebook():?

Reply:
Oh...apology... In my test env, I added 'not', forgot that for the pushing...